test/raw_data_gis/scripts/test_raster_bio1clip.json

```diff
@@ -0,0 +1,49 @@
+{
+    "archived": "zip",
+    "citation": "",
+    "dataset_count": "1",
+    "datum": "",
+    "description": "bio1clip.tif",
+    "driver": "GTiff",
+    "file_size": "",
+    "group_count": "",
+    "homepage": "fill",
+    "keywords": [],
+    "licenses": [],
+    "name": "test-raster-bio1clip",
+    "projection": "GEOGCS[\"WGS 84\",DATUM[\"WGS_1984\",SPHEROID[\"WGS 84\",6378137,298.257223563,AUTHORITY[\"EPSG\",\"7030\"]],AUTHORITY[\"EPSG\",\"6326\"]],PRIMEM[\"Greenwich\",0],UNIT[\"degree\",0.0174532925199433,AUTHORITY[\"EPSG\",\"9122\"]],AXIS[\"Latitude\",NORTH],AXIS[\"Longitude\",EAST],AUTHORITY[\"EPSG\",\"4326\"]]",
+    "resources": [
+        {
+            "band_name": "bio1clip_1",
+            "color_table": null,
+            "extensions": [
+                "tif"
+            ],
+            "format": "raster",
+            "name": "bio1clip",
+            "no_data_value": -9999.0,
+            "other_paths": "",
+            "path": "bio1clip.tif",
```

Comment:
This might be what causes the error in finding the file. So I did some commands using `unzip`, to see how files are stored in the archive. And I compared the `test_raster_bio1clip.zip` to `test_raster_bio1.zip` (which is present in the current version of retriever).

![terminal](https://user-images.githubusercontent.com/44918472/126823563-84e86370-e753-490a-80d6-80dd72773a8c.png)

In the image, you can clearly see that the `tif` file is present in the folder `test_raster_bio1clip` in the `test_raster_bio1clip.zip`, while for the `test_raster_bio1.zip`, the file is present in the root of the archive.

So either you change the archive, or change the path in the script

Reply:
@henrykironde